app/src/main/java/com/nononsenseapps/feeder/ui/compose/feed/FeedScreen.kt

```diff
@@ -495,145 +509,393 @@ fun FeedScreen(
+                                                false -> Icons.Default.CheckBoxOutlineBlank
+                                            },
+                                            contentDescription = null,
+                                        )
+                                    },
+                                    text = {
+                                        Text(stringResource(id = R.string.read_adjective))
+                                    },
+                                    modifier =
+                                        Modifier
+                                            .safeSemantics {
+                                                stateDescription =
+                                                    when (viewState.filter.read) {
+                                                        true -> context.getString(androidx.compose.ui.R.string.selected)
+                                                        else -> context.getString(androidx.compose.ui.R.string.not_selected)
+                                                    }
+                                                role = Role.Checkbox
+                                            },
+                                )
+                                // Hidden button for TalkBack
+                                DropdownMenuItem(
+                                    onClick = {
+                                        onShowFilterMenu(false)
+                                    },
+                                    text = {},
+                                    modifier =
+                                        Modifier
+                                            .height(0.dp)
+                                            .safeSemantics {
+                                                contentDescription = closeMenuText
+                                                role = Role.Button
+                                            },
+                                )
+                            }
+                        }
+                    }
+                }
+
+                PlainTooltipBox(tooltip = { Text(stringResource(R.string.search_noun)) }) {
                     Box {
                         IconButton(
-                            onClick = { onShowFilterMenu(true) },
+                            onClick = { onShowSearchBar(true) },
                         ) {
                             Icon(
-                                Icons.Default.FilterList,
-                                contentDescription = stringResource(R.string.filter_noun),
+                                Icons.Default.Search,
+                                contentDescription = stringResource(R.string.search_noun),
+                                tint = if (viewState.filter.search.isEmpty()) androidx.compose.material3.LocalContentColor.current else MaterialTheme.colorScheme.primary,
```

Comment:
Subtle to change the icon color when a search filter is applied. Probably too subtle... I'm leaning towards ensuring the search bar can't be closed when non-empty (or without being cleared)